### PR TITLE
Keep class and function names during minimization

### PR DESCRIPTION
**Pull request by alalamav**
- Preserves class and function names during TS/JS code minimization in order to enable more accurate debugging in error reports.
- Other minimization settings remain the same, since webpack uses `TerserPlugin` with a default configuration in production mode.

### CODE REVIEW

File: src/www/base.webpack.js [hunk 1]
```diff
@@ -47,6 +48,14 @@ exports.makeConfig = (options) => {
       extensions: ['.tsx', '.ts', '.js'],
       fallback: {'url': require.resolve('url/')},
     },
+    optimization: {
+      minimizer: [new TerserPlugin({
+        terserOptions: {
+          keep_classnames: true,
```
Comment (fortuna) [inline src/www/base.webpack.js:54]: That will probably make the minimization significantly less efficient.
I wonder if we should restrict to specific types or files.

File: src/www/base.webpack.js [hunk 2]
```diff
@@ -47,6 +48,14 @@ exports.makeConfig = (options) => {
...
+          keep_fnames: true,
```
Comment (fortuna) [inline src/www/base.webpack.js:55]: Why is this needed?

**Review by fortuna — APPROVED** [review]

File: src/www/base.webpack.js (hunk 1 above)
Comment (alalamav) [reply src/www/base.webpack.js:54]: I compared the minified file size and it only differs by 200KB with this change.

File: src/www/base.webpack.js (hunk 2 above)
Comment (alalamav) [reply src/www/base.webpack.js:55]: Our logging code relies on `Class.prototype.name` so we need to preserve function names in addition to class names. See the [terser docs](https://github.com/terser/terser#minify-options) for more information.

**Review by fortuna — APPROVED** [review]
